manager/controllers/utils.go

```diff
@@ -25,7 +25,7 @@ func LogAndRespNotFound(c *gin.Context, err error, respMsg string) {
 	c.JSON(http.StatusNotFound, utils.ErrorResponse{Error: respMsg})
 }
 
-func ApplySort(c *gin.Context, tx *gorm.DB, allowedFields ...string) (*gorm.DB, error) {
+func ApplySort(c *gin.Context, tx *gorm.DB, tablePrefix string, allowedFields ...string) (*gorm.DB, error) {
```

Comment:
I would remove this tablePrefix feature. Let's suppose no prefix. We can solve it using AS keyword on query level. This solution does not allow usage of columns from two tables (two different prefixes) in case of join anyway. What do you mean @beav ?

Reply:
That would be better solution. But it does not work when there are 2 tables in a join clause which both have sort attribute.